lib/components/term.js

```diff
@@ -98,9 +106,24 @@ export default class Term extends Component {
 
     this.getScreenNode().addEventListener('mouseup', this.handleMouseUp);
     this.getScreenNode().addEventListener('mousedown', this.handleMouseDown);
+    this.getScreenNode().addEventListener('contextmenu', this.handleRightClick);
     terms[this.props.uid] = this;
   }
 
+  handleRightClick(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    const {quickEdit} = config.getConfig();
```

Comment:
You dont have to import/check `config`. You should check `this.props.quickEdit` like  [here](https://github.com/zeit/hyper/blob/5ec705000f73dbad5f70b9152708cbeaf4e3c086/lib/components/term.js#L283)